src/org/ods/scheduler/LeVADocumentScheduler.groovy

```diff
@@ -126,9 +126,13 @@ class LeVADocumentScheduler extends DocGenScheduler {
 
     // Document types per environment type token (MROPipelineUtil.getBuildParams().targetEnvironmentToken)
     // Only Q and P, in D all types are generated
-    private static Map ENVIRONMENT_TYPE = [
+    public static Map ENVIRONMENT_TYPE = [
```

Comment:
@hrcornejo currently, the list is not complete and contains documents that should not be contained. Let's discuss soon.

Reply:
Updated list